### ISSUE TITLE
Wrong detection for redundant param statement

### ISSUE THREAD

**Issue by Aetet** [jscs-dev/jscs-jsdoc#79]
if I have params with properties and more than one argument I have wrong detection for redundant param statement.

Example of code linting fails on jscs-jsdoc v 0.4.0:
```javascript
/**
 * Example function
 * @param {Object} options some options
 * @param {Number} options.id some id
 * @param {String} options.filename some filename
 * @param {Array} rest rest arguments
 * @returns {Boolean} to be or not to be
 */
function hello(options, rest) {
	return (options.filename) ? true : false;
}
```

Fail log:
```
Message:
    redundant param statement at example.js :
     1 |/**
--------^
     2 | * Example function
     3 | * @param {Object} options some options
```

If code looks like this it works like a charm:
```javascript
/**
 * Example function
 * @param {Object} options some options
 * @param {Number} options.id some id
 * @param {String} options.filename some filename
 * @returns {Boolean} to be or not to be
 */
function hello(options) {
	return (options.filename) ? true : false;
}
```

**Comment by zxqfox** [comment]
Ouch. I didn't notice that ;-( Thanks for a report!

**Comment by Aetet** [comment]
You are welcome.

**Comment by zxqfox** [comment]
Thanks again. Landed in v0.4.1
